src/pushsource/_impl/model/ami.py

```diff
@@ -41,6 +41,17 @@ class AmiRelease(object):
     """Release type, typically "ga" or "beta"."""
 
 
+@attr.s()
+class AmiBillingCodes(object):
+    """Billing codes associated with an AMI."""
+
+    name = attr.ib(type=str, default=None, validator=instance_of_str)
+    """Billing codes name, for example: Hourly2, arbitrary string for making image name unique"""
```

Comment:
Minor: the existing doc string style is to end the doc string with a ".", making it more of a full sentence rather than sentence fragment.  It'd be best to stick with that.